flang/test/Semantics/OpenMP/implicit-dsa.f90

```diff
@@ -169,3 +169,39 @@ subroutine implicit_dsa_test8
     end do
   !$omp end task
 end subroutine
+
+! Test variables defined in modules default to shared DSA
+!DEF: /implicit_dsa_test9_mod Module
+module implicit_dsa_test9_mod
+ !DEF: /implicit_dsa_test9_mod/tm3a PUBLIC (InDataStmt) ObjectEntity COMPLEX(4)
+  complex tm3a/(0,0)/
```

Comment:
nit: add a module variable that doesn't have an initializer and reference it in the task below.